### PR TITLE
Made thermostat history graph localizable

### PR DESCRIPTION
**Pull request by springstan**
Issues affected: #3952

Changes:
- added 4 key-value pairs to src/translations/en.json
- updated src/components/state-history-chart-line.js to use translation instead of static string
- joined multiple strings together, separated by a white space 
 -> `[name, target_temperature].join(" ")` = 'Ecobee target temperature'

Results:
other
range

### CODE REVIEW

**Comment by bramkragten** [comment]
Use template literals instead of `join`:
```
[name, this.hass.localize("ui.card.climate.heating")].join(" ")
 => 
`${name} ${this.hass.localize("ui.card.climate.heating")}`
```

And we should pass `name` as a variable to the translation string.

File: src/components/state-history-chart-line.js [hunk 1]
```diff
@@ -181,23 +181,65 @@ class StateHistoryChartLine extends LocalizeMixin(PolymerElement) {
...
+            this.hass
+              .localize("ui.card.climate.current_temperature")
+              .toLowerCase(),
+          ].join(" "),
+          true
+        );
         if (hasHeat) {
-          addColumn(name + " heating", true, true);
+          addColumn(
+            [name, this.hass.localize("ui.card.climate.heating")].join(" "),
```
Comment (bramkragten) [inline src/components/state-history-chart-line.js]:
```suggestion
            `${name} ${this.hass.localize("ui.card.climate.heating")}`,
```

Comment (springstan) [reply src/components/state-history-chart-line.js]: Yeah you are completely right, that simplifies it immensely 👍

Comment (springstan) [reply src/components/state-history-chart-line.js]: > And we should pass name as a variable to the translation string.

@bramkragten so both or just template literals?

Comment (bramkragten) [reply src/components/state-history-chart-line.js]: Can't do both ;-) just add it as a variable, that's nicer.

Comment (springstan) [reply src/components/state-history-chart-line.js]: Yeah I agree ;-) sure thing.

**Review by springstan — COMMENTED** [review]
@bramkragten are my fixes with  https://github.com/home-assistant/home-assistant-polymer/pull/4004/commits/5453dd0030fdad4d94c2d76b375a7caa8bd4b961 okay?

File: src/components/state-history-chart-line.js [hunk 2]
```diff
@@ -181,23 +181,65 @@ class StateHistoryChartLine extends LocalizeMixin(PolymerElement) {
...
-          addColumn(name + " target temperature high", true);
-          addColumn(name + " target temperature low", true);
+          addColumn(
+            [
+              name,
+              this.hass
+                .localize("ui.card.climate.target_temperature")
+                .toLowerCase(),
+              this.hass.localize("ui.card.climate.high"),
+            ].join(" "),
```
[the remark this answers is not in the document]
Comment (bramkragten) [reply src/components/state-history-chart-line.js]: Make it 2 translation keys: `{name} current temperature` and `{name} current temperature {mode}`

**Comment by bramkragten** [comment]
> @bramkragten are my fixes with [5453dd0](https://github.com/home-assistant/home-assistant-polymer/commit/5453dd0030fdad4d94c2d76b375a7caa8bd4b961) okay?

I ment using the variables in the translations, so not to use the template literals. That way translators can move the words around if needed.

File: src/components/state-history-chart-line.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (springstan) [reply src/components/state-history-chart-line.js]: > Make it 2 translation keys: `{name} current temperature` and `{name} current temperature {mode}`
That would definitely be a better solution, I wil do that.

[the remark this answers is not in the document]
Comment (springstan) [reply src/components/state-history-chart-line.js]: @bramkragten Fixed with be3dc3d745521d19c3574b6ba981c87534825979 :)

File: src/translations/en.json [hunk 3]
```diff
@@ -402,6 +402,13 @@
         "currently": "Currently",
         "on_off": "On / off",
         "target_temperature": "Target temperature",
+        "target_temperature_entity": "{name} target temperature",
+        "target_temperature_mode": "{name} target temperature {mode}",
+        "current_temperature": "{name} current temperature",
+        "heating": " {name} heating",
```
Comment (bramkragten) [inline src/translations/en.json]:
```suggestion
        "heating": "{name} heating",
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**